Makefile.custom.mk

```diff
@@ -0,0 +1,65 @@
+.PHONY: ensure-gcloud-exists
+ensure-gcloud-exists:
+ifeq  (, $(shell which gcloud))
+	$(error "gcloud not found. Please install and initialize. Installation instructions found here - https://cloud.google.com/sdk/docs/install")
+endif
+
+.PHONY: ensure-gcp-envs
+ensure-gcp-envs:
+ifndef GCP_PROJECT_ID
+	$(error GCP_PROJECT_ID is undefined)
+endif
+
+.PHONY: all
+all: capg-firewall-rule-operator capg-operator cluster-control-plane-node cluster-worker-node csi-driver-app dns-operator-gcp external-dns-app fleet-membership-operator-gcp
```

Comment:
Sorry, my OCD kicked in: why `capg-operator` instead of `cluster-api-provider-gcp-app`?